### PR TITLE
Add Notification type

### PR DESCRIPTION
**Pull request by wcandillon**
Minor improvement to the typing declaration.

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -24,6 +24,19 @@ declare module "react-native-fcm" {
         const Local = "local_notification";
     }
 
+    export interface Notification {
+        collapse_key: string;
+        opened_from_tray: boolean;
+        from: string;
+        notification: {
+            body: string;
+            e: string;
```
Comment (evollu) [inline index.d.ts]: what is e?

**Comment by wcandillon** [comment]
@evollu: Sorry about that, I fixed it!

**Comment by wcandillon** [comment]
@evollu Thanks so much for the review. I have pushed the improvements.

**Comment by evollu** [comment]
how about this
```
  export interface Notification {
        collapse_key: string;
        opened_from_tray: boolean;
        from: string;
        notification: {
            title?: string
            body: string;
            icon: string;
        };
        _notificationType: string;
        finish(type?: string): void;
    }

    export interface LocalNotification {
        title?: string;
        body: string;
        icon?: string;
        vibrate?: number;
        sound?: boolean;
        big_text?: string;
        large_icon?: string;
        priority?: string
    }

    export interface ScheduleLocalNotification extends LocalNotification{
        id: string;
        fire_date: number
    }
```

**Comment by wcandillon** [comment]
@evollu oh woah nice catches, thank you so much! I updated the PR.

**Pull request merged**